### DISCUSSION TITLE
photoview in Terra-Master

### DISCUSSION THREAD

**Discussion by mariolav** [photoview/photoview#904] [Q&A and Troubleshooting]
Hi, I was able to get photoview to work on my F2-423 Terra-Master, but I have no access to the root id, so I don't see the docker folder, but i found a docker folder under here /home/docker/photoview/photos
I also created the db folder and cache folder but nothing ever gets added to those folders
I am using this and it works
- /home/docker/photoview/photos:/photos2:ro

but then I added another folder where i already had some photos:
 - /PhotosVideosPublic/photoview/photos:/photos3:ro
and added that folder in photoview settings, but scan does not act on it.
What am I doing wrong. I am new to anything Linux, so pardon my ignorance.

**Comment by mariolav** [comment]
Sorry, I should have said that I found a folder named docker here:
/home/docker
and I created the folders /home/docker/photoview/photos, /home/docker/photoview/db and /home/docker/photoview/cache

**Comment by kkovaletp** [comment]
> F2-423 Terra-Master

What is this? What distro of Linux do you use? If this is some restricted\hardened environment, please describe the limitations

Also, try to use the setup, described in my PR #863 - it may solve your issue. But clean up all leftovers of the Photoview on your system prior to that, including the docker containers and volumes.
